### PR TITLE
Refs #12970 - Package autoprefixer-rails 5.2.1.3 for Patternfly

### PR DESCRIPTION
**Pull request by dLobatog**
https://github.com/theforeman/foreman-packaging/pull/944#issue-124499878

### CODE REVIEW

File: rel-eng/tito.props [hunk 1]
```diff
@@ -118,6 +118,7 @@ whitelist = foreman
   foreman-selinux
   rubygem-ace-rails-ap
   rubygem-activerecord-session_store
+  rubygem-autoprefixer-rails
```
Comment (domcleal) [inline rel-eng/tito.props]: Nitpick, move this down as it's alphabetical.

File: rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec [hunk 2]
```diff
@@ -0,0 +1,77 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name autoprefixer-rails
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 5.2.1.3
+Release: 1%{?dist}
+Summary: Parse CSS and add vendor prefixes to CSS rules using values from the Can I Use website
```
Comment (domcleal) [inline rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec]: Nitpick, trim this to less than 72 characters.

File: rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Group: Development/Languages
+License: MIT
+URL: https://github.com/ai/autoprefixer-rails
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ror}rubygem(execjs) => 0
```
Comment (domcleal) [inline rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec]: Nitpick, no need for `=> 0` on these two lines.

File: rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.rspec
```
Comment (domcleal) [inline rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec]: Exclude this (or `.*`) or move to -doc, it's not needed at runtime.

File: rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+%doc %{gem_instdir}/CHANGELOG.md
+%{gem_instdir}/Gemfile
+%{gem_instdir}/Gemfile.rails3
+%doc %{gem_instdir}/README.md
+%{gem_instdir}/Rakefile
+%{gem_instdir}/autoprefixer-rails.gemspec
+%{gem_instdir}/spec
+
+%changelog
+* Tue Jan 05 2016 Daniel Lobato - 5.2.1.3-1
```
Comment (domcleal) [inline rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec]: missing e-mail address

**Comment by domcleal** [comment]
1) Missing `ruby(release)` and `rubygems` runtime Requires.

**Comment by dLobatog** [comment]
@domcleal Updated according to your comments, thanks!

File: foreman/foreman.spec [hunk 6]
```diff
@@ -190,6 +190,8 @@ BuildRequires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 BuildRequires: %{?scl_prefix_ror}rubygem(execjs) < 2.5.0
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+BuildRequires: %{?scl_prefix_ror}rubygem(autoprefixer-rails) >= 5.2
```
Comment (domcleal) [inline foreman/foreman.spec]: a) scl_prefix as it's in our collection
b) layout should match assets.rb, move below jquery-ui-rails

**Comment by domcleal** [comment]
Thanks @dLobatog, building now.

**Pull request merged**